kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/flight_finder/FlightFinder.java

```diff
@@ -1,35 +1,83 @@
 package com.kodilla.good.patterns.flight_finder;
 
-import java.util.List;
+import java.util.*;
```

Comment:
nie używaj importu z * - jest to postrzegane jako zła praktyka.
https://stackoverflow.com/a/147461

Reply:
Nie przypominam sobie, bym to tworzył to "wina" IJ